engines/bastion_katello/test/ostree-branches/details/ostree-branches-details-repositories.controller.test.js

```diff
@@ -0,0 +1,63 @@
+describe('Controller: OstreeBranchesDetailsRepositoriesController', function() {
+    var $scope, translate, Nutupane, ContentView, Environment, Repository,
+        CurrentOrganization;
+
+    beforeEach(module('Bastion.ostree-branches', 'Bastion.test-mocks'));
+
+    beforeEach(inject(function($injector) {
+        var $controller = $injector.get('$controller'),
+            MockResource = $injector.get('MockResource'),
+            $q = $injector.get('$q');
+
+
+        translate = function (string) {
+            return string;
+        };
+
+        Nutupane = function() {
+            this.table = {
+                params: {},
+                showColumns: function () {},
+                allResultsSelectCount: function () {}
+            };
+            this.enableSelectAllResults = function () {};
+            this.getAllSelectedResults = function () {
+                return {include: [1, 2, 3]};
+            };
+            this.setParams = function (params) {
+                this.table.params = params;
+            };
+            this.refresh = function () {};
+            this.load = function () {
+                return {then: function () {}}
+            };
+            this.setSearchKey = function () {};
+        };
+
+        Environment = MockResource.$new();
+        Environment.mockResources = {results: [{id: 5, library: true}]};
+
+        ContentView = MockResource.$new();
+        Repository = MockResource.$new();
+
+        CurrentOrganization = 'foo';
+
+        $scope = $injector.get('$rootScope').$new();
+        $scope.$stateParams = {branchId: 1};
+
+        $controller('OstreeBranchesDetailsRepositoriesController', {
+            $scope: $scope,
+            $q: $q,
+            translate: translate,
+            Nutupane: Nutupane,
+            ContentView: ContentView,
+            Repository: Repository,
+            Environment: Environment,
+            CurrentOrganization: CurrentOrganization
+        });
+    }));
+
+    it("puts the ostree branches repositories table object on the scope", function() {
+        expect($scope.detailsTable).toBeDefined();
+    });
```

Comment:
This controller does a good deal more than you give it credit for in this test.  At the very least you need a test for `$scope.filterBranches`.